app/scripts/modules/kubernetes/container/configurer.directive.html

```diff
@@ -32,6 +32,46 @@
     </div>
   </div>
 
+  <collapsible-section heading="Basic Settings" expanded="false">
+    <div class="form-group">
+      <div class="sm-label-left">
+        <b>Environment Variables</b>
+      </div>
+      <table class="table table-condensed">
+        <thead>
+        <tr>
+          <th>Name</th>
+          <th>Value</th>
+          <th></th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr ng-repeat="pair in container.envVars">
```

Comment:
Yeah, I saw that, but in the future the env var attribute will have a third field, namely one for secrets passed in via Kubernetes. The API already supports it, so when I add it I don't want to have to refactor.